src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -637,14 +634,15 @@ static Peep* viewport_interaction_get_closest_peep(ScreenCoordsXY screenCoords,
     screenCoords.x = ((screenCoords.x - viewport->pos.x) * viewport->zoom) + viewport->viewPos.x;
     screenCoords.y = ((screenCoords.y - viewport->pos.y) * viewport->zoom) + viewport->viewPos.y;
 
-    closestPeep = nullptr;
-    closestDistance = 0xFFFF;
-    FOR_ALL_PEEPS (spriteIndex, peep)
+    Peep* closestPeep = nullptr;
+    auto closestDistance = std::numeric_limits<int32_t>::max();
```

Comment:
This used to be `std::numeric_limits<uint16_t>::max();`

Reply:
It doesn't really matter what it is as long as it's the biggest value for the type. I prefer we standardise towards int32 for basic variables.

Reply:
Are you sure its callers never check for 0xFFFF?

Reply:
Its only used in this one function so it is safe to change.